core-war/src/main/java/org/silverpeas/web/jobdomain/servlets/JobDomainPeasRequestRouter.java

```diff
@@ -545,70 +542,64 @@ else if (function.startsWith("domainCreate")) {
           newGroup.setSuperGroupId(request.getParameter("Idgroup"));
           request.setAttribute("groupObject", newGroup);
           request.setAttribute("action", "groupCreate");
-          request.setAttribute("groupsPath", jobDomainSC.getPath(
-              (String) request.getAttribute("myComponentURL"), jobDomainSC
-              .getString("JDP.groupAdd")
-              + "..."));
+          request.setAttribute("groupsPath", jobDomainSC.getPath((String) request.getAttribute("myComponentURL"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "groupsPath" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)